### DISCUSSION TITLE
Github Account Recovery

### DISCUSSION THREAD

**Discussion by sovitshrestha** [community/community#33006] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello, I've lost the access to my github account completely. While trying to perform reset or forgot password process I am asked to either use recovery codes or request a reset. Both of the options do not work in my case since I have lost the access to TOTP authenticator app, also my laptop has been formatted. I don't find any other way to recover my github account. This is my friends account using to get help from github community. So if anyone could help me with this issue, it would be a great help...

**Comment by airtower-luna** [comment, marked as answer]
You can see the options you have in the documentation article [Recover an account with 2FA](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials). If none of those work you can still try to [contact GitHub support](https://support.github.com/request), but they might not be able help, as the documentation warns:

> For security reasons, GitHub Support may not be able to restore access to accounts with two-factor authentication enabled if you lose your two-factor authentication credentials or lose access to your account recovery methods.

Good luck!

**Reply by sovitshrestha** [reply]
I lost my recovery codes, as well as access to authenticator application since my phone was formatted and here I forgot about authenticator app. Then Is there no way to recover it back?

**Reply by Ambatrichard29** [reply]
I lost my phone number

**Reply by sevvalozdamar** [reply]
Hi, I have the same problem. Have you solved?

**Comment by Jasper-1010** [comment]
I don't understand all this, I've signed into a new account with Github, because I can't remember how I signed in to old one. I changed mobile numbers, And recently Microsoft, made a change, from Micro Office, to Microsoft 365  Nothing seems to work properly since.  Authenticator won't give me codes any longer to access facebook.  Nothing transferred over from old phone to my authenticator app.  I'm completely locked out of facebook now; as it refuses to send txt msgs since I've changed my number, and I can't find any old recovery codes for facebook.  I've had trouble since I received my new phone, and mobile number in early September 2022; I'm to a point now where I'm stuck, all passwords and code authenticator codes, are lost.  Somehow I ended up with a katnimmons.katnimmons@microsoft.com account, not sure how that occurred but I'm sure I followed some sort of directions online that led to this new account with github.  I'd love to return to kat@katnimmons.com repository original github, and somehow recover my authenticator codes, so I can again get into my accounts.  HELP!

**Comment by Jasper-1010** [comment]
I don't understand all this, I've signed into a new account with Github, because I can't remember how I signed in to old one. I changed mobile numbers, And recently Microsoft, made a change, from Micro Office, to Microsoft 365  Nothing seems to work properly since.  Authenticator won't give me codes any longer to access facebook.  Nothing transferred over from old phone to my authenticator app.  I'm completely locked out of facebook now; as it refuses to send txt msgs since I've changed my number, and I can't find any old recovery codes for facebook.  I've had trouble since.

**Reply by Jasper-1010** [reply]
again, doing this on my own, is just not possible, as I don't even understand the instructions  to get verified recoverys

**Comment by afurgeri** [comment]
Hi, I need help.

I lost access to my account, AlejoLespaul, it has two-factor authentication, and I have no way to regain access.

I had to make another account, but I wish I could get my old account back.

**Reply by Mahidlaskar** [reply]
Yes

**Reply by cornel-pe** [reply]
have you recover your old account?

**Reply by sivatejachary** [reply]
tell to me how to recover the old account

**Comment by myasir5532** [comment]
![Uploading Screenshot_2024-09-28-22-44-09-44_ee1cec40dcf6eb3919ecbfc5d87e6719.jpg…]()

**Comment by thomassiard** [comment]
Hello!

I'm sorry to hear about your situation. Losing access to your GitHub account can be frustrating, especially with TOTP authentication. Here are a few steps you can try to regain access:

Contact GitHub Support:

The best course of action is to reach out directly to [GitHub Support](https://support.github.com/contact). Explain your situation in detail, including that you have lost access to your TOTP authenticator app and the recovery codes.
Use Backup Codes:

If you have previously saved any backup codes provided during the setup of your two-factor authentication (2FA), you can use those to regain access.
Account Recovery Request:

In the support request, make sure to include any relevant information about your account (like username and email associated with it) to verify your identity.
Check Email for Any Alerts:

Look for any email from GitHub that might offer account recovery options. Sometimes they may send instructions for recovery.
Revisit TOTP App:

If you can, check if you have a backup of your TOTP app or if you can access it through another device where it might be installed.
If you follow these steps, hopefully, GitHub Support can assist you in recovering your account. Good luck, and I hope you regain access soon!

**Comment by Abhijais4896** [comment]
#[Using a two-factor authentication recovery code](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials#using-a-two-factor-authentication-recovery-code)
#[Authenticating with a passkey](https://docs.github.com[Authenticating with a verified device, SSH token, or personal access token](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials#authenticating-with-a-verified-device-ssh-token-or-personal-access-token)/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials#authenticating-with-a-passkey)
#[Authenticating with a verified device, SSH token, or personal access token](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials#authenticating-with-a-verified-device-ssh-token-or-personal-access-token)

**Comment by NXakimov** [comment]
Dear GitHub Support Team,

I am reaching out regarding an issue with accessing my GitHub account. I had enabled two-factor authentication (2FA) using Google Authenticator, but unfortunately, I lost my phone along with access to my iCloud backup.

Since I no longer have access to my 2FA codes, I am unable to log into my GitHub account. I would like to request an account recovery process so that I can regain access. Here are some details that might help verify my ownership:

- **GitHub Username**: [Xakimov1610]
- **Registered Email**: [nosirbekxakimov01@gmail.com]
- **Any other information that might help verify my identity**

I understand that security is crucial, and I am willing to provide any additional verification if required. Please guide me on the next steps for recovering my account.

Thank you for your support.

Best regards,  
[Xakimov Nosirbek]

**Reply by sevvalozdamar** [reply]
Hi, I have the same problem. Have you solved?

**Comment by ebadifardin** [comment]
Hello, good morning. I lost my account, but the only thing that is possible to recover and to prove that this account is mine is that the authenticator program is still active. I wanted to know if there is any hope for recovery and that I have a lot of time. I sent a message to the support but I did not receive a response.

**Reply by aybangueco** [reply]
I lost my account too. Looks like we won't be able to recover it after all.

**Comment by badofdad** [comment]
Subject: Urgent: GitHub Account Recovery – Lost 2FA & Primary Email

Hello GitHub Support,

I have lost access to my GitHub account and need assistance recovering it. The details are as follows:

GitHub Username: eSportxD

Primary Email: (Lost access)

Secondary Email: hexorit3+2@gmail.com (I still have access)

Issue: I lost my 2FA authentication codes and recovery codes, and I no longer have access to my primary email.

I can provide any necessary verification details, such as repository names, past commits, or billing details, if needed. Please let me know how I can regain access to my account.

Thank you for your help.

Best regards,
Kanna
